universum/configuration_support.py

```diff
@@ -8,14 +8,133 @@
     "Step",
     "Configuration",
     "Variations",
+    "combine",
     "set_project_root",
     "get_project_root"
 ]
 
 
+# pylint: disable-msg=line-too-long
 class Step:
     """
-    Step is a collection of configurable project-specific data needed for Universum operations.
+    Step is a collection of configurable project-specific data entries needed for Universum functionality. Individual
+    build steps are collected in a :class:`Configuration` object, which supports arithmetic operations for
+    combination and chaining.
```

Comment:
Is 'chaining' an arithmetic operation? I thought it was 'combination and addition'? If 'chaining' is more correct, we should fix the 'configuring' paragraph name as well.

Reply:
chaining is not an arithmetic operation, but that's what really happens when you multiple configurations (via children).